test/benchmarks/dhrystone/dhrystone.c

```diff
@@ -0,0 +1,185 @@
+// See LICENSE for license details.
+
+#pragma GCC optimize ("no-inline")
```

Comment:
@ccelio that's what we do in riscv-tests

Reply:
````
diff --git a/test/benchmarks/dhrystone/check b/test/benchmarks/dhrystone/check
index 79f5456ec4be..4a828f36debf 100755
--- a/test/benchmarks/dhrystone/check
+++ b/test/benchmarks/dhrystone/check
@@ -28,7 +28,7 @@ echo "ERROR: $march-$mabi failed to run" >$out
 
 tempdir=$(mktemp -d)
 trap "rm -rf $tempdir" EXIT
-$cc ${c[@]} -march=$march -mabi=$mabi -O3 -o $tempdir/dhrystone -static -Wno-all
+$cc ${c[@]} -march=$march -mabi=$mabi -O3 -fno-common -fno-inline -o $tempdir/dhrystone -static -Wno-all
 
 $objdump -d $tempdir/dhrystone > ~/dump
 begin_pc=$($objdump -d $tempdir/dhrystone | grep 'Begin_Time' | grep -e 'sd' -e 'sw' | cut -d: -f1 | xargs echo)
````

that didn't make any difference, so I think it's probably safe

Reply:
I see, I didn't notice that. Is `no-common` required as well for a "valid" run?

Reply:
no-common isn't required; it just speeds it up on RISC-V (more auipcs/luis become gp-relative accesses).